build.sbt

```diff
@@ -34,16 +34,16 @@ def javacOptionsVersion(scalaVersion: String): Seq[String] = {
 
 // Provide a managed dependency on X if -DXVersion="" is supplied on the command line.
 val defaultVersions = Map(
-  "chisel3" -> "3.2-SNAPSHOT",
-  "chisel-iotesters" -> "1.3-SNAPSHOT",
-  "rocketchip" -> "1.2-SNAPSHOT"
+  "chisel3" -> "3.3-SNAPSHOT",
```

Comment:
Does this need to be here, doesn't it get the right chisel from the testers dependency

Reply:
It should (get the correct chisel3 version from the testers' dependency). I think it's more of a "documentation" issue than an implementation one. We certainly could delete this. It would make it easier for experts to manage their dependencies. Let's see what @grebe thinks.

Reply:
I feel like it's probably better to delete.